### PR TITLE
Add new core team members

### PR DESCRIPTION
**Pull request by fabpot**
As announced here: https://symfony.com/blog/new-core-team-members

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

File: contributing/code/core_team.rst
Comment (jderusse) [inline contributing/code/core_team.rst]:
```suggestion
  * **Jérémy Derussé** (`jderusse`_).
```

File: contributing/code/core_team.rst
Comment (jderusse) [inline contributing/code/core_team.rst]:
```suggestion
  * **Jérémy Derussé** (`jderusse`_).
```

**Review by jderusse — COMMENTED** [review]
misspelled `jderusse`

**Review by derrabus — APPROVED** [review]

**Review by jderusse — APPROVED** [review]

**Review by Nyholm — APPROVED** [review]
Thank you

**Review by OskarStark — APPROVED** [review]

**Comment by wouterj** [comment]
It's a bit weird to merge this PR myself, but let's do it after so many approvals. I'm happy to have join the code team as well now! :)

**Pull request merged**
